src/game_initialization/multiplayer_configure.cpp

```diff
@@ -264,7 +269,10 @@ configure::~configure()
 	// don't set observers preference if disabled (for singleplayer)
 	if (observers_game_.enabled())
 		preferences::set_allow_observers(engine_.allow_observers());
-
+	// don't set registerd_users_only preference if disabled (for singleplayer)
+	if (registerd_users_only_.enabled())
```

Comment:
It's minor, sure, but I dislike having misspelled variable names (the proper spelling is "registered").